sirepo/auth/email.py

```diff
@@ -173,3 +185,14 @@ def avatar_uri(qcall, model, size):
         hashlib.md5(pykern.pkcompat.to_bytes(model.user_name)).hexdigest(),
         size,
     )
+
+
+def init_apis(*args, **kwargs):
+    global _cfg
+    _cfg = pkconfig.init(
+        deny_access_domains=(
+            {"gmail.com", "yahoo.com", "outlook.com"},
```

Comment:
I don't think we should have any addresses by default. Also, `set` is sufficient for the parser. It always creates a `frozenset`.

Reply:
34cc6ad44